htdocs/contact/list.php

```diff
@@ -435,6 +452,8 @@
 $parameters = array();
 $reshook = $hookmanager->executeHooks('printFieldListWhere', $parameters); // Note that $action and $object may have been modified by hook
 $sql .= $hookmanager->resPrint;
+$sql .= ' GROUP BY p.rowid';
```

Comment:
Why a group by here if it is on a rowid ? There is no grouping function in requets. It seems you use the group by to avoid the duplicate due to left join on categories (a side effect of group by when forgetting the grouping function). if this is the case, adding a DISTINCT if we add filter on categories looks a more logical hoice

Reply:
I don't understand the problem, group by is used on product list

Reply:
?

Reply:
@eldy ?

Reply:
@eldy

Reply:
@eldy

Reply:
@eldy

Reply:
@eldy

Reply:
A group by must be used when there is a mathematical operation into select (a sum, a count, ...). Here there is no grouping mathematical operation into the select so why adding a critera to say how to group the mathematical operation.

Reply:
Ok, so you would like me to use subquery instead of group by to handle multi line  problem due to join categ? 
In product list there is no sum or count but the group by is present.